helper/keysutil/transit_lru.go

```diff
@@ -3,12 +3,13 @@ package keysutil
 import lru "github.com/hashicorp/golang-lru"
 
 type TransitLRU struct {
-	lru *lru.TwoQueueCache
+	size int
+	lru  *lru.TwoQueueCache
 }
 
 func NewTransitLRU(size int) *TransitLRU {
 	lru, _ := lru.New2Q(size)
```

Comment:
Why do you ignore the error value?

Reply:
shouldn't be ignored imo - it bubbles up to the user now.